robottelo/api/utils.py

```diff
@@ -946,3 +946,26 @@ def skip_yum_update_during_provisioning(template=None, reverse=False):
         return update_provisioning_template(name=template, old=old, new=new)
     else:
         return update_provisioning_template(name=template, old=new, new=old)
+
+
+def hammer_api_timeout_set(timeout=-1, reverse=False):
```

Comment:
I would have kept the function name as 'set_hammer_api_timeout'.

Reply:
done